frontend/src/metabase/querying/components/FilterPicker/FilterPickerBody/FilterPickerBody.tsx

```diff
@@ -60,7 +60,7 @@ function getFilterWidget(column: Lib.ColumnMetadata) {
   if (Lib.isNumeric(column)) {
     return NumberFilterPicker;
   }
-  if (Lib.isString(column)) {
+  if (Lib.isString(column) || Lib.isStringLike(column)) {
```

Comment:
You also need to update
- `stringFilterParts` https://github.com/metabase/metabase/blob/4b71e9aa645117a944af409882bda33f64225bf9/frontend/src/metabase-lib/filter.ts#L139
- `ColumnFilterSection` https://github.com/metabase/metabase/blob/2aa3cec610588238132afbc789da0845c7c2ec5d/frontend/src/metabase/querying/components/FilterModal/ColumnFilterSection/ColumnFilterSection.tsx#L64

Reply:
Maybe we should export a function `isStringOrStringLike` from `column_types.ts` if it becomes a common operation.